### PR TITLE
1.1.0rc0

### PR DESCRIPTION
**Pull request by swcurran**
Ready for 1.1.0rc0.

- Normal publication PR -- updating references, generating and documenting the changelog -- please review the changelog!
- Fixed a few references to markdown files that changed.
- The "acapy_agent" change resulted in the Read The Docs source files changing -- hundreds of files deleted/added.
- Slight improvements to the scripts and instructions for publishing releases

Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
 # Aries Cloud Agent Python Changelog
 
+## 1.1.0rc0
+
+### October 11, 2024
+
+Release 1.0.2 is the first release of ACA-Py from the [OpenWallet Foundation] (OWF). The only reason for the release is to test out all of the release publishing actions now that we have moved the repo to its new home ([https://github.com/openwallet-foundation/acapy](https://github.com/openwallet-foundation/acapy)). Almost all of the changes in the release are related to the move.
```
Comment (jamshale) [inline CHANGELOG.md:7]: 1.1.0 **

Comment (swcurran) [reply CHANGELOG.md]: Doh. Fixed.

**Review by jamshale — APPROVED** [review]
Description looked good to me 👍

**Review by esune — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**